core/src/lib.rs

```diff
@@ -16,6 +16,3 @@ pub mod consts;
 pub mod subaddress;
 
 pub mod slip10;
-
-#[cfg(feature = "protos")]
-pub mod protos;
```

Comment:
Why remove this? Should we also delete `protos.rs`?

Reply:
yep yep, i added it when looking at reworking some encodings for the hw wallet, but we're not continuing down that particular path